### PR TITLE
Fix scrolling issue on mobile devices.

### LINKED ISSUE #56: Cannot scroll non-mobile view if mobile menu is open
When opening the menu on a mobile viewport and widening the viewport past the mobile breakpoint, the user is no longer able to scroll.

### PR DESCRIPTION
**Pull request by vernondegoede**
Fixes #56.

The mobile menu currently prevents scrolling because it's on top of the main content area. By hiding the entire navigation, scrolling is possible on mobile devices again.

### CODE REVIEW

**Comment by vernondegoede** [comment]
@mattijsbliek Thanks. The utility class will be removed and the mobile navigation will be closed if it was previously opened and the user resized to desktop viewport.

File: source/theme/js/mobileNavigationButton.js [hunk 1]
```diff
@@ -10,4 +14,24 @@ export default enhance("mobile-navigation-button", () => {
...
+        if (documentWidth > MOBILE_NAV_BREAKPOINT) {
+          documentElement.classList.remove(NO_SCROLL_UTILITY_CLASS);
+          mobileSidebar.classList.remove("is-visible");
+          navigationTrigger.classList.remove("is-opened");
+        }
+      });
+    }
+  };
+
+  window.addEventListener("resize", debounce(() => allowScrollingOnDesktop(), 200, false));
```
Comment (mattijsbliek) [inline source/theme/js/mobileNavigationButton.js]: I think you can simplify this to:

```js
window.addEventListener("resize", debounce(allowScrollingOnDesktop, 200));
```

Comment (vernondegoede) [reply source/theme/js/mobileNavigationButton.js]: You're right, thanks!

File: source/theme/js/mobileNavigationButton.js [hunk 2]
```diff
@@ -10,4 +14,24 @@ export default enhance("mobile-navigation-button", () => {
     navigationTrigger.classList.toggle("is-opened");
     document.documentElement.classList.toggle("u-no-scroll");
```
[the remark this answers is not in the document]
Comment (vernondegoede) [reply source/theme/js/mobileNavigationButton.js]: 👀

**Comment by vernondegoede** [comment]
@mattijsbliek Any additional feedback? I'd like to merge this one ASAP as the documentation is currently not usable on mobile devices.

**Comment by mattijsbliek** [comment]
Nope, ship it!

**Pull request merged**